lib/git.rb

```diff
@@ -123,12 +123,12 @@ def self.delete_current_branch()
 
       Git::switch_branch(devBranch)
 
-      `git branch -d #{branch}`.strip
+      `git branch -d #{branch.shellescape}`.strip
    end
 
    # Returns the SHA1 hash that the specified branch or symbol points to
    def self.branch_hash(branch)
-      `git rev-parse --verify --quiet "#{branch}" 2>/dev/null`.strip
+      `git rev-parse --verify --quiet "#{branch.shellescape}" 2>/dev/null`.strip
```

Comment:
This is escaping the `{` and `}` in `"#{current}@{upstream}"`

Reply:
which causes feature pull to fail. Do we want another function called upstream_branch_hash that only escapes the branch name?